be/src/storage/schema_change.cpp

```diff
@@ -853,6 +854,11 @@ RowBlockAllocator::~RowBlockAllocator() {
     }
 }
 
+bool RowBlockAllocator::can_sort(size_t num_rows, size_t allocated_rows) {
```

Comment:
IMO, `can_sort` is not a suitable name.
This function returns a state which may lead to a number of possibilities, of which whether to perform `SORT` or not is only one option.
So it is better to have a name that clearly describes the state returned by the function, rather than the action taken based on the state